src/CaptureClient/CaptureClient.cpp

```diff
@@ -140,11 +139,8 @@ ErrorMessageOr<CaptureListener::CaptureOutcome> CaptureClient::CaptureSync(
   } else {
     capture_options->set_samples_per_second(samples_per_second);
     capture_options->set_stack_dump_size(stack_dump_size);
-    if (unwinding_method == UnwindingMethod::kFramePointerUnwinding) {
-      capture_options->set_unwinding_method(CaptureOptions::kFramePointers);
-    } else {
-      capture_options->set_unwinding_method(CaptureOptions::kDwarf);
-    }
+    CHECK(unwinding_method != CaptureOptions::kUndefined);
```

Comment:
Hmm, I am unsure whether I like the check here, or whether I would prefer an "if undefined then Dwarf". Leaving this to you.

Reply:
Really shouldn't happen since I sanitized the input. Leaving as is.